rust/tw_bitcoin/src/ffi/scripts.rs

```diff
@@ -146,15 +146,20 @@ pub unsafe extern "C" fn tw_build_brc20_transfer_inscription(
 
 #[no_mangle]
 // Builds the Ordinals inscripton for BRC20 transfer.
-pub unsafe extern "C" fn tw_build_nft_inscription(
-    mime_type: u32,
+pub unsafe extern "C" fn tw_bitcoin_build_nft_inscription(
+    mime_type: *const u8,
+    mime_type_len: usize,
     data: *const u8,
     data_len: usize,
     satoshis: i64,
     pubkey: *const u8,
     pubkey_len: usize,
 ) -> CByteArray {
-    let mime_type = try_or_else!(MimeType::from_raw(mime_type as usize), CByteArray::null);
+    // Convert mimeType.
+    let mime_type = try_or_else!(
+        CByteArrayRef::new(mime_type, mime_type_len).as_slice(),
```

Comment:
Should `mime_type` be a string?
An example
https://github.com/trustwallet/wallet-core/blob/f266567e913a40bfee80b8ecdd4b74785cea2b32/rust/tw_encoding/src/ffi.rs#L97-L100

But I plan to implement `TWString` same way as in C++